chore: update native-lend to v2 TVLs

June-native opened the pull request:
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama): 
Native V2

##### Twitter Link:
https://x.com/native_fi

##### List of audit links if any:
https://docs.native.org/native-dev/resources/audits

##### Website Link:
https://native.org/

##### Logo (High resolution, will be shown with rounded borders):
https://docs.native.org/native-dev/resources/media-kit

##### Current TVL:
700k

##### Treasury Addresses (if the protocol has treasury)


##### Chain:
Ethereum, Bsc, Arbitrum, Base

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)


##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)


##### Short Description (to be shown on DefiLlama):
Native is an on-chain platform to build token liquidity that is openly accessible and cost effective. It serves as an alternative to traditional AMMs through integration of two innovative designs: the [Native Swap Engine](https://docs.native.org/native-dev/solution/native-swap-engine) and [Native Credit Pool](https://docs.native.org/native-dev/solution/native-credit-pool).

##### Token address and ticker if any:


##### Category (full list at https://defillama.com/categories) *Please choose only one:
Lending

##### Oracle Provider(s): Specify the oracle(s) used (e.g., Chainlink, Band, API3, TWAP, etc.):
##### Implementation Details: Briefly describe how the oracle is integrated into your project:
##### Documentation/Proof: Provide links to documentation or any other resources that verify the oracle's usage:

##### forkedFrom (Does your project originate from another project):


##### methodology (what is being counted as tvl, how is tvl being calculated):
Total amount of underlying assets in each deployed LP tokens/vaults.

##### Github org/user (Optional, if your code is open source, we can track activity):
http://github.com/native-org

waynebruce0x commented on projects/native-lend/index.js:
```diff
@@ -1,26 +1,53 @@
+      // get all underlying tokens
+      const tokens = await api.multiCall({ abi: 'address:underlying', calls: lps });
+
+      // get total supplied, not cash
```
`tvl` of lending protocols should be available liquidity. Please export any borrows as `borrows` so that users can choose to include them in TVL stats

June-native commented:
Hi:
- updated `tvl` to use token cash balance in vault
- added `borrowed`, calc by `locked > cash ? lock - cash : 0` . Since our vault LPs can not only be shorted (borrowed) but also longed.

Please kindly check.

waynebruce0x commented:
Does this produce expected results when you test locally? I see the CI is failing to get BSC logs

June-native commented:
> Does this produce expected results when you test locally? I see the CI is failing to get BSC logs

Yes it does:

```
node test.js projects/native-lend/index.js
Building prices get queries for 2 tokens
Building prices get queries for 12 tokens
Building prices get queries for 2 tokens
Building prices get queries for 3 tokens
Building prices get queries for 19 tokens
--- base-borrowed ---
Total: 0 

--- arbitrum-borrowed ---
Total: 0 

--- bsc-borrowed ---
SKATE                     2.9124259982082663e-8
Total: 2.9124259982082663e-8 

--- bsc ---
USDT                      103.45 k
dodo                      18.75 k
WBNB                      15.88 k
PARTI                     15.63 k
SHELL                     8.27 k
RDO                       403.60891696271705
BANK                      108.5505397257756
SIGN                      55.48730860227497
SOON                      26.983520845078104
DOGE                      9.814534421232262
ETH                       8.970110892018088
HAEDAL                    8.888944294046249
USDC                      1.9495785545210793
BTCB                      1.1897651806896619
SAHARA                    1.1493825830455489
SKATE                     0.042857710875740015
Total: 162.61 k 

--- ethereum ---
WETH                      203.76 k
USDT                      191.93 k
STO                       45.91 k
USDC                      40.69 k
WUSD                      16.00 k
RDO                       15.02 k
WBTC                      567.8042348
SOPH                      21.245476270002754
DODO                      0.23512451041088797
PLUME                     0.12841507711880837
SKATE                     0.0428570235382676
Total: 513.89 k 

--- base ---
WETH                      489.5648760309872
USDC                      489.10361307849803
Total: 978.6684891094852 

--- ethereum-borrowed ---
USDC                      159.32 k
STO                       1.36 k
SKATE                     7.164617323978477e-7
Total: 160.67 k 

--- arbitrum ---
Total: 0 

--- borrowed ---
USDC                      159.31 k
STO                       1.36 k
Total: 160.67 k 

--- tvl ---
USDT                      295.38 k
WETH                      204.25 k
STO                       45.91 k
USDC                      41.18 k
dodo                      18.75 k
WUSD                      16.00 k
WBNB                      15.88 k
PARTI                     15.63 k
RDO                       15.42 k
SHELL                     8.27 k
WBTC                      568
BANK                      109
SIGN                      55
SOON                      27
SOPH                      21
DOGE                      10
ETH                       9
HAEDAL                    9
BTCB                      1
SAHARA                    1
Total: 677.48 k 

------ TVL ------
ethereum                  513.89 k
bsc                       162.61 k
borrowed                  160.67 k
ethereum-borrowed         160.67 k
base                      979
bsc-borrowed              0
base-borrowed             0
arbitrum-borrowed         0
arbitrum                  0

total                    677.48 k 
```

From the logs it appears to be some RPC related issues when trying to `getLogs`
